### PR TITLE
Added previously unsupported xml based filetypes to the code snippet …

### PR DESCRIPTION
**Pull request by adunndevster**
…extension

### CODE REVIEW

**Comment by adunndevster** [comment]
@928PJY - Could you please take a look at this?

**Comment by 928PJY** [comment]
@adunndevster sure, @yufeih to review as well.

File: src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ public class HtmlCodeSnippetRenderer : HtmlObjectRenderer<CodeSnippet>
             { "batchfile", new string[] {"bat", "cmd" } },
             { "css", Array.Empty<string>() },
             { "cpp", new string[] {"c", "c++", "objective-c", "obj-c", "objc", "objectivec", "h", "hpp", "cc", "m" } },
-            { "csharp", new string[] {"cs"} },
+            { "csharp", new string[] {"cs", "asax", "fx"} },
```
[the remark this answers is not in the document]
Comment (adunndevster) [reply src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs]: Great catch! Thank you!

**Review by yufeih — APPROVED** [review]

**Comment by yufeih** [comment]
Oops, you may want to target v3 branch

**Comment by adunndevster** [comment]
@yufeih, thanks! It's been a while!

**Pull request closed without merging**